.gitignore

```diff
@@ -27,7 +27,7 @@ cover.html
 build/.out
 
 # Node modules
-internal/nginx/modules/node_modules
+node_modules/
```

Comment:
I think we still want to include line 30 for local development.

Reply:
`node_modules/` will ignore the folder wherever it is in the tree